### ISSUE TITLE
Unrecognized documented parameter enable_default_security for security groups

### ISSUE THREAD

**Issue by sblaisot** [scaleway/terraform-provider-scaleway#726]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.24
+ provider.scaleway v1.17.2

### Affected Resource(s)

* scaleway_instance_security_group

### Terraform Configuration Files

```hcl
resource "scaleway_security_group" "test" {
  name                    = "test"
  description             = "test"
  enable_default_security = true
  stateful                = true
  inbound_default_policy  = "accept"
  outbound_default_policy = "drop"
}
```

### Output

```
Error: Unsupported argument

  on main.tf line 38, in resource "scaleway_instance_security_group" "secgroup":
  38:   enable_default_security = "false"

An argument named "enable_default_security" is not expected here.
```

### Expected Behavior

Security group is created and default security rules are not created (outgoing SMTP is permitted)

### Actual Behavior

Unrecognized `enable_default_security` parameter despite [it's documented](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/security_group#enable_default_security)
fail to create security group with default secuity disabled (allowing outgoing SMTP)

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

There is no such option in the code: https://github.com/scaleway/terraform-provider-scaleway/blob/master/scaleway/resource_instance_security_group.go#L27

**Comment by sblaisot** [comment]
I was on the wrong documentation and looked for a deprecated object.

This option existed in `scaleway_security_group` but doesn't exist anymore in `scaleway_instance_security_group`

What is the correct way to create a security group allowing SMTP output?

**Comment by remyleone** [comment]
Hello, the first step is to get approval for sending SMTP traffic from the Scaleway support team. Once this is done, you can choose to set `enable_default_security` to false to be able to send the SMTP traffic. 
`scaleway_instance_security_group` does support `enable_default_security_group` ([code](https://github.com/scaleway/terraform-provider-scaleway/blob/master/scaleway/resource_instance_security_group.go#L85) in the upcoming v2 release.

**Comment by remyleone** [comment]
Support was merged in: https://github.com/scaleway/terraform-provider-scaleway/pull/651 Feel free to open a new issue if other problems arise :)
